src/components/messages-view-configurator/MessagesViewConfigurator.tsx

```diff
@@ -18,6 +18,7 @@ import React, { useRef, useState } from 'react';
 import { useOutsideClickListener } from '../../hooks';
 import { ModalPortal } from '../util/Portal';
 import '../../styles/messages-view-configurator.scss';
+import '../../styles/messages.scss';
```

Comment:
there are too many imports of messages.scss. i think we can delete all these imports and use import in MessagesWindow